src/components/Home.jsx

```diff
@@ -0,0 +1,60 @@
+import { BallTriangle } from 'react-loader-spinner';
+import { useSelector } from 'react-redux';
+import '../styles/home.css';
+import Match from './Match';
+
+export default function Home() {
+  const { games, inCommingGames } = useSelector((store) => store.games);
+  let color = false;
+
+  const createMatch = (item, col) => {
+    color = col ? !color : color;
+    return (
+      <Match
+        key={item.id}
+        data={item}
+        team={false}
+        odd={color}
+      />
+    );
+  };
+
+  if (!games || !inCommingGames) {
+    return (
+      <div className="loading-screen">
+        <BallTriangle
+          height={150}
+          width={150}
+          radius={5}
+          color="#712041"
+          ariaLabel="ball-triangle-loading"
+          visible
+        />
+      </div>
+    );
+  }
+  return (
+    <>
+      <section>
+        <h1 className="subtitle">Last matches</h1>
+        <div className="matches-list">
+          {
+            games.map((item, i) => (
+              createMatch(item, (i + 1) % 2 === 0, i)
+            ))
+          }
+        </div>
+      </section>
+      <section>
+        <h1 className="subtitle">Comming matches</h1>
+        <div className="matches-list">
+          {
+            inCommingGames.map((item, i) => (
+              createMatch(item, (i + 1) % 2 === 0, i)
+            ))
+          }
+        </div>
+      </section>
```

Comment:
- [ ] From the project requirements, on the home page, you were supposed to have something like a search functionality that can help users filter the contents displayed on the home page based on his/her search. Here is a screenshot taken from the project requirements 
![image](https://user-images.githubusercontent.com/90222110/226131106-f6eac989-2d71-4b40-9a1a-cb4209a9d857.png)

For your project specifically, I think you need to implement the search functionality that allows users to search for a team and then display the `last matches` of that specific team that the user searched for. That is what they are referring to as "displaying data that could be filtered based on parameter" Please implement a search or any filter functionality that will allow the user to see the `last matches` or the `coming matches` of the team that he/she searched and these will be displayed either on a new page or just still on the home page. It will depend on your liking. 😃 👍 
Guides:
You may use the following project as a reference to see how the filter functionality can be implemented 👉 [here](https://github.com/BertrandConxy/Stocks-market-trends)
And this medium article 👉 [here](https://javascript.plainenglish.io/how-to-build-user-search-app-in-react-5bf1eff63657)

Reply:
Sorry my links were commented. Thanks for all your clever recommendations i just want to say that there is actually a category filter ( team filter) on the top left menu. that is part of the main component. I fixed all the issues, thank you so much.